test/util/e2e.go

```diff
@@ -259,7 +259,10 @@ func WaitForActivePodsAndTerminate(ctx context.Context, k8sClient client.Client,
 		g.Expect(k8sClient.List(ctx, &pods, podListOpts)).To(gomega.Succeed())
 		activePods = make([]corev1.Pod, 0)
 		for _, p := range pods.Items {
-			if (len(p.Status.PodIP) != 0 && p.Status.PodIP != "0.0.0.0") && (p.Status.Phase == corev1.PodRunning || p.Status.Phase == corev1.PodPending) {
+			if len(p.Status.PodIP) != 0 && p.Status.Phase == corev1.PodRunning {
+				cmd := []string{"/bin/sh", "-c", fmt.Sprintf("curl \"http://%s:8080/readyz\"", p.Status.PodIP)}
+				_, _, err := KExecute(ctx, cfg, restClient, namespace, p.Name, p.Spec.Containers[0].Name, cmd)
+				gomega.ExpectWithOffset(1, err).ToNot(gomega.HaveOccurred())
```

Comment:
```suggestion
				g.ExpectWithOffset(1, err).ToNot(gomega.HaveOccurred())
```
probably would work too?

Reply:
Sure, I have changed that, works